### PR TITLE
issue1494: fix missing parameter bindings

### LINKED ISSUE #1494: AixLib contains models with experiment annotation that have parameters without bindings and without explicitly set start values.
Dear AixLib developers,

please check the latest OpenModelica [regression test report](https://libraries.openmodelica.org/branches/history/master/2024-03-06%2022:42:24..2024-03-07%2022:25:06.html). For the latest released version of AixLib ([AixLib](http://libraries.openmodelica.org/branches/master/AixLib/AixLib.html)) and for the development version of AixLib ([AixLib_dev](http://libraries.openmodelica.org/branches/master/AixLib/AixLib.html)) there are about 35 models that fail since we fixed ticket OpenModelica/OpenModelica#10386.

According to the latest version of Modelica Specification, i.e., 3.6, when you simulate a model all parameters with `fixed = true` should have a binding equation (either provided as a default in the components, or set by modifiers when instantiating them). The only exception is parameters that have an explicitly set start attribute - in that case, if a binding equation is lacking, the start attribute is used to set the parameter value and a warning should be issued.

The first failing case is [AixLib.Electrical.Machines.PVInverterRMS](http://libraries.openmodelica.org/branches/master/AixLib/files/AixLib_AixLib.Electrical.Machines.PVInverterRMS.err), which is actually a component, not a model that should be simulated. In this case it is OK not to have a binding for some parameters, as they will be provided when instantiating the component, but you should remove the `experiment` annotation, since it makes no sense to simulate this component by itself.

In all other cases, some parameter bindings are missing. You can
- add a default binding to the parameters in the component definitions, or
- add an explicit binding when you instantiate the components or,
- add an explicit start = 0 attribute to the parameters in the component definition, which will be used as a fallback value, with a warning being issued.

This will make those models compliant with the Modelica Specification in force, and get them running again in OpenModelica.

Thanks!

### PR DESCRIPTION
**Pull request by jonas0902**
closes #1494 
Missing parameter bindings have been added in the form of default parameters.
Redundant conditional arguments in a non-connect context have been removed.

### CODE REVIEW

File: AixLib/Obsolete/YearIndependent/FastHVAC/Components/Chiller/Chiller.mo [hunk 1]
```diff
@@ -226,12 +226,6 @@ model Chiller "Base model of FastHVAC Chiller"
         extent={{6,-6},{-6,6}},
         rotation=90,
         origin={16,-52})));
-  Modelica.Blocks.Interfaces.RealInput iceFac_in if not useBusConnectorOnly
```
Comment (FWuellhorst) [inline AixLib/Obsolete/YearIndependent/FastHVAC/Components/Chiller/Chiller.mo:229]: Please revert this change, this should not impact OM

File: AixLib/Obsolete/YearIndependent/FastHVAC/Examples/Chiller/Chiller.mo [hunk 2]
```diff
@@ -123,6 +123,9 @@ model Chiller
     annotation (Placement(transformation(extent={{46,-82},{34,-70}})));
   Modelica.Blocks.Logical.LogicalSwitch logicalSwitch
     annotation (Placement(transformation(extent={{10,-78},{0,-88}})));
+  Controls.Interfaces.VapourCompressionMachineControlBus sigBus1
+    annotation (Placement(transformation(extent={{18,-28},{30,-16}}),
+        iconTransformation(extent={{18,-28},{30,-16}})));
```
Comment (FWuellhorst) [inline AixLib/Obsolete/YearIndependent/FastHVAC/Examples/Chiller/Chiller.mo:128]: Why was this necessary? Obsolete models may not work in either case, but I don't get why this concept does not work in OM

**Review by FWuellhorst — COMMENTED** [review]
Looks good, only two minor questions!

File: AixLib/Obsolete/YearIndependent/FastHVAC/Examples/Chiller/Chiller.mo (hunk 2 above)
Comment (jonas0902) [reply AixLib/Obsolete/YearIndependent/FastHVAC/Examples/Chiller/Chiller.mo:128]: There was an error in Dymola and OM because the chiller example was underdetermined. 
According to the error message the sigBus needed an input.

**Comment by FWuellhorst** [comment]
@jonas0902 : Can you check the model `Error: Check Model AixLib.ThermalZones.HighOrder.Validation.EmpiricalValidation.Warehouse`? Error:
```
     Error:     AixLib.ThermalZones.HighOrder.Validation.EmpiricalValidation.Warehouse  
['Check of <a href="Modelica://AixLib.ThermalZones.HighOrder.Validation.EmpiricalValidation.Warehouse">AixLib.ThermalZones.HighOrder.Validation.EmpiricalValidation.Warehouse</a>:\nError: The following parameters don\'t have any value:\n  room.absInnerWallSurf\n  room.coeffTableSolDistrFractions.coeffCeiling\n  room.coeffTableSolDistrFractions.coeffFloor\n  room.coeffTableSolDistrFractions.coeffOWEast\n  room.coeffTableSolDistrFractions.coeffOWNorth\n  room.coeffTableSolDistrFractions.coeffOWSouth\n  room.coeffTableSolDistrFractions.coeffOWWest\n  room.coeffTableSolDistrFractions.coeffWinAbs\n  room.coeffTableSolDistrFractions.coeffWinLost\n\nThe model has the same number of unknowns and equations: 1161\nThe model has the same number of unknowns and equations\nfor the given numerical settings of parameters: 1161\nError: ERRORS have been issued.\n', 2, 0, 3]
```

**Review by FWuellhorst — APPROVED** [review]
CI passes, thanks for the fix!
